### ISSUE TITLE
Priority names not correct

### ISSUE THREAD

**Issue by matsbrucke** [wolfgangimig/itol#3]
Hi,

the plugin does not pull the rigth names for priorities from redmine. We use "niedrig, normal, wichtig, eilig, sofort". The plugin offers "normal, high prio, urgent, immediate"

best

Matthias

**Comment by wolfgangimig** [comment]
This issue is similar to #2.

The list of priorities is defined at:

```
var propPriority = propertyClasses.get(Property.PRIORITY);
propPriority.setSelectList([ new IdName(1, "Low priority"),
        new IdName(2, "Normal priority"), new IdName(3, "High priority"),
        new IdName(4, "Urgent"), new IdName(5, "Immediate") ]);
```

**Comment by wolfgangimig** [comment]
Solved in version 1.1
